engine/release-notes/index.md

```diff
@@ -22,6 +22,66 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.4
+2021-02-26
+
+### Builder
+
+* Fix incorrect cache match for inline cache with empty layers [moby/moby#42061](https://github.com/moby/moby/pull/42061)
+* Update BuildKit to v0.8.2 [moby/moby#42061](https://github.com/moby/moby/pull/42061)
+  * Fix seccomp compatibility in 32bit arm allowing to build arm/v7 on arm64 machines
+  * resolver: avoid error caching on token fetch
+  * fileop: fix checksum to contain indexes of inputs preventing certain cache misses
+  * Update qemu emulators. Fixes a networking regression
```

Comment:
no impact on moby

Reply:
What networking regression btw?

Reply:
something with traceroute https://github.com/docker/buildx/issues/528#issuecomment-774070063

Reply:
This is completely unrelated. Embedded emulators were updated **to** 5.2, that is the version this user claims **has** the regression. But again, these are not used in moby anyway.